easybuild/easyblocks/n/numpy.py

```diff
@@ -244,8 +245,11 @@ def sanity_check_step(self, *args, **kwargs):
         }
         custom_commands = [
             ('python', '-c "import numpy"'),
-            ('python', '-c "import numpy.core._dotblas"'),  # _dotblas is required for decent performance of numpy.dot()
         ]
+        if LooseVersion(self.version) < LooseVersion("1.10"):
+            custom_commands.append (
+                ('python', '-c "import numpy.core._dotblas"')
+            )  # _dotblas is required for decent performance of numpy.dot()
```

Comment:
if you want to avoid too long lines, just put the command above the `append`:

```python
# _dotblas is required for decent performance of numpy.dot()
custom_commands.append(('python', '-c "import numpy.core._dotblas"'))
```